lib/heapLogger.js

```diff
@@ -0,0 +1,86 @@
+// (c) Copyright 2021, SAP SE and ClearlyDefined contributors. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
```

Comment:
Thanks for the contribution from Microsoft!  An example header see, https://github.com/clearlydefined/service/blob/f79a63c5d32dfe00c423bff25cf254fe754ecd82/lib/utils.js#L1